node/tests/TestUtilities.ts

```diff
@@ -1137,18 +1211,29 @@ export async function transactionTest(
                 3,
             ]);
             baseTransaction.zinter([key27, key26]);
-            responseData.push(["zinter([key27, key26])", ["one", "two"]]);
+            responseData.push([
+                "zinter([key27, key26])",
+                [
+                    toStringReturn("one", decoder),
+                    toStringReturn("two", decoder),
+                ],
+            ]);
```

Comment:
I'm confused about these (and other) changes in this file. Your changes don't update all of these commands (such as zinter, zdiff). Is this just a case where we missed testing binary APIs for these commands?

Reply:
I have reverted most of these changes to the return. 
I was going too far with these tests.  We DO need to convert the arguments, but we will only test String returns here for transactions.